### PR TITLE
Allow including extra properties in the client

### PR DESCRIPTION
**Pull request by jaapromijn**
Per @gisostallenberg's request, reviving https://github.com/ConnectHolland/sulu-block-bundle/pull/18

### CODE REVIEW

**Comment by gisostallenberg** [comment]
I would like to see this applied to all properties applied in blocks from the start.

**Comment by gisostallenberg** [comment]
Can you please fix the commit itself, so it is your own, with your own words.

**Comment by reyostallenberg** [comment]
Why reference internal projects in an open library?

File: src/Resources/templates/blocks/images.xml [hunk 1]
```diff
@@ -10,8 +10,13 @@
         <title lang="en">Images</title>
         <title lang="nl">Afbeeldingen</title>
     </meta>
+
     <properties>
         <xi:include href='sulu-block-bundle://properties/title.xml'/>
         <xi:include href='sulu-block-bundle://properties/images.xml'/>
+        <xi:include href='sulu-block-bundle://app-properties/images.xml' parse="xml">
```
Comment (gisostallenberg) [inline src/Resources/templates/blocks/images.xml]: Is ` parse="xml"` needed? If so, please use single quotes.

**Comment by reyostallenberg** [comment]
It would be nice to document this feature by giving an example of whet file (example content) to add on what path

File: README.md [hunk 2]
```diff
@@ -130,3 +130,27 @@ For example like this:
...
+<!-- app/Resources/ConnectHollandSuluBlockBundle/templates/propreties/images.xml -->
+<?xml version='1.0' ?>
+<properties xmlns='http://schemas.sulu.io/template/template'
+    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
+    xsi:schemaLocation='http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd'
+    >
+    <property
+        name='caption'
+        type='text_line'
+
```
Comment (reyostallenberg) [inline README.md]: Why is this whitespace, I even think it's better to put all attributes on line 144

**Pull request merged**